tasks/src/Zero-shot-classification/data.ts

```diff
@@ -0,0 +1,30 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "",
+			id:          "",
+		},
+		{
+			description: "",
+			id:          "",
+		},
+	],
+	
+	models: [
+		{
+			description: "",
+			id:          "",
+		},
+		{
+			description: "",
+			id:          "i",

```

Comment:
```suggestion
			id:          "",

```
I think this is a mistake right?